src/game/middlewares/phoenix-solution-middleware.js

```diff
@@ -29,6 +29,11 @@ export const phoenixSolutionMiddleware = state => next => action => {
         store.dispatch(selectionAction);
     }
 
+    if (message.name === MESSAGE_NAME.puzzleChanged) {
+        const clearSelectorAction = clearSelector();
```

Comment:
Maybe call this one directly?

Reply:
I prefer having separate var for easier debug. Also it looks more consistent with previous case, where one-line call will look not so pretty

Reply:
👍